.github/actions/run-fixtures-test/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Run a fixtures test
+description: Run a fixtures test
+
+inputs:
+  fixtures-dir:
+    description: Path to the fixtures directory
+    required: true
+  command:
+    description: The git-cliff command to run
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: nightly
+        override: true
+
+    - name: Install git-cliff
+      run: |
+        cargo install --path git-cliff/
+      shell: bash
+
+    - name: Set git config
+      run: |
+        git config --global user.name github-actions
+        git config --global user.email github-actions@github.com
+      shell: bash
+
+    - name: Create commits and tags
+      working-directory: ${{ inputs.fixtures-dir }}
+      run: |
+        git init
+        ./commit.sh
+      shell: bash
+
+    - name: Generate a changelog
+      working-directory: ${{ inputs.fixtures-dir }}
+      run: |
+        git cliff --config cliff.toml ${{ inputs.command }} > output.md
```

Comment:
nitpick: one-line?

Reply:
I'm sorry, what did you mean? :thinking:

Reply:
Ah, to combine with https://github.com/orhun/git-cliff/pull/58#discussion_r800243359?

Reply:
Ah, no, like this?

```yaml
      run: git cliff --config cliff.toml ${{ inputs.command }} > output.md
```

Reply:
To minimize the diff when we add lines later, I prefer always writing `|`, but I'll adapt to your style.
Will wait for a reply comment.

Reply:
> ```yaml
>       run: git cliff --config cliff.toml ${{ inputs.command }} > output.md
> ```

Yup that's what I meant. Same thing as https://github.com/orhun/git-cliff/pull/58#discussion_r800243359

Reply:
Okay, but just out of curiosity, can I ask the reason? To keep consistency?